### PR TITLE
Feature: service status

### PR DESCRIPTION
**Pull request by luixo**
I decided not to add a debug version of switches for testing purposes, any developer can do this by himself.

### CODE REVIEW

**Comment by luixo** [comment]
@shelegdmitriy 
Ready for review

File: backend/src/types.ts [hunk 1]
```diff
@@ -5,3 +5,13 @@ export type CurrentEpochState = {
   totalStake: string;
   height: number;
 };
+
+export type HealthStatus = { timestamp: number } & (
+  | {
+      ok: true;
```
Comment (shelegdmitriy) [inline backend/src/types.ts:11]: Why `true` and not `boolean`?

File: frontend/src/components/utils/ServiceStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+import { MINUTE, SECOND } from "../../libraries/time";
+import { HealthStatus } from "../../types/common";
+import Timer from "./Timer";
+
+const Wrapper = styled("div", {
+  height: "100%",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+
```
Comment (shelegdmitriy) [inline frontend/src/components/utils/ServiceStatus.tsx:19]: ?

Comment (luixo) [reply frontend/src/components/utils/ServiceStatus.tsx:19]: It's the same as having an empty space between some logical parts of code, I guess :)

**Pull request merged**